### PR TITLE
Support dump and load bytes and add back up limit

### PR DESCRIPTION
**Pull request by fatanugraha**
1. Added support to back up and restore bytes data.
2. Added new flags to stop the backup process after reached certain limit.

### CODE REVIEW

File: dynamodump/dynamodump.py [hunk 1]
```diff
@@ -49,10 +49,25 @@
...
+import base64
 
+def encoder(self, obj):
+    if isinstance(obj, datetime.datetime):
+        return obj.isoformat()
+
+    if isinstance(obj, bytes):
+        return base64.b64encode(obj).decode("utf-8")
+
+    return json.JSONEncoder.encoder(self, ob)
```
Comment (asimerel) [inline dynamodump/dynamodump.py:61]:
```suggestion
    return json.JSONEncoder.encoder(self, obj)
```
I think there is a typo here.

Comment (fatanugraha) [reply dynamodump/dynamodump.py]: Thanks for pointing this out, not sure why this passed my manual testing in the past 🤔

**Review by bchew — COMMENTED** [review]
@fatanugraha thanks for the PR! Could you please resolve the linting issue picked up by CI 🙂

File: dynamodump/dynamodump.py [hunk 2]
```diff
@@ -1241,6 +1263,9 @@ def main():
     parser.add_argument(
         "--log", help="Logging level - DEBUG|INFO|WARNING|ERROR|CRITICAL [optional]"
     )
+    parser.add_argument(
+        "--limit", help="Limit option for backup, will stop the back up process after number of backed up items exceeded the limit [optional]", type=int
```
[the remark this answers is not in the document]
Comment (fatanugraha) [reply dynamodump/dynamodump.py]: Done!

**Comment by fatanugraha** [comment]
Ah apologies, I missed out flake8 lint issues. Pushed the change 🙇

**Review by bchew — APPROVED** [review]
Thanks @fatanugraha

**Pull request merged**